ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -72,12 +72,9 @@ class SimpleOperationTracker implements OperationTracker {
   private static final Logger logger = LoggerFactory.getLogger(SimpleOperationTracker.class);
   protected final String datacenterName;
   protected final String originatingDcName;
-  protected final int diskReplicaSuccessTarget;
-  protected final int diskReplicaParallelism;
-  protected final int cloudReplicaSuccessTarget;
-  protected final int cloudReplicaParallelism;
-  protected final boolean cloudReplicasPresent;
-  protected final boolean diskReplicasPresent;
+  protected final int replicaSuccessTarget;
+  protected final int replicaParallelism;
+  protected final boolean replicasPresent;
```

Comment:
we probably don't need this variable anymore. The replica will be present otherwise we would fail from the constructor.

Reply:
Done in 
212d358